packages/espree/lib/options.js

```diff
@@ -22,19 +26,30 @@ const SUPPORTED_VERSIONS = [
     15, // 2024
     16, // 2025
     17 // 2026
-];
+]);
+
+/**
+ * @typedef {typeof SUPPORTED_VERSIONS[number]} normalizedEcmaVersion
+ */
+
+const LATEST_ECMA_VERSION =
+    /* eslint-disable jsdoc/valid-types -- Bug */
+    /** @type {typeof SUPPORTED_VERSIONS extends readonly [...infer _R, infer L] ? L : never} */ (
```

Comment:
```suggestion
    /** @type {typeof SUPPORTED_VERSIONS extends readonly [...unknown[], infer L] ? L : never} */ (
```

since `_R` is unused.